CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.4.5] - 2019-12-22
+
 ### Changed
 - The k8s host id does not use the "{@account}:host:conjur/authn-k8s/#{@service_name}/apps"
   prefix and takes the full host-id from the CSR. We also handle backwards-compatibility and use
```

Comment:
Minor nitpick but "We also handle backwards-compatibility and use..." could be part of a new ### Added section because we are adding backwards compatibility

Reply:
i don't agree here. The backwards compatibility is part of the change and doesn't have enough context outside of it.